### PR TITLE
Create Namespace Discussion 2023-09-18.md

### PR DESCRIPTION
**Pull request by robcraig-LF**
Adds meeting minutes and agenda from Namespace Discussion 2023-09-18

### CODE REVIEW

File: tech/Namespace Discussion 2023-09-18.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# Namespace Discussion 2023-09-18
+
+## Attendees
+* Gary O'Neall
+* David Kemp
+* Sean Barnum
+* Bob
```
Comment (goneall) [inline tech/Namespace Discussion 2023-09-18.md:7]:
```suggestion
* Bob Martin
```

File: tech/Namespace Discussion 2023-09-18.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
+# Namespace Discussion 2023-09-18
+
+## Attendees
+* Gary O'Neall
+* David Kemp
+* Sean Barnum
+* Bob 
+
+# Minutes
```
Comment (goneall) [inline tech/Namespace Discussion 2023-09-18.md:9]:
```suggestion
```

**Review by goneall — CHANGES_REQUESTED** [review]
Thanks @robcraig-LF for posting this - if you could make the 2 minor corrections.

**Comment by robcraig-LF** [comment]
Updates made as requested

**Comment by goneall** [comment]
@robcraig-LF - Thanks for the update.  I don't think you intended to close the pull request (perhaps you intended to close the comment issue)?  

I'll re-open and merge.

**Review by goneall — APPROVED** [review]
LGTM

**Pull request merged**